Common/Orders/OrderEvent.cs

```diff
@@ -111,6 +121,25 @@ public OrderEvent(Order order, string message = "")
         }
 
         /// <summary>
+        /// Helper Constructor for TradeBuilder tests
+        /// </summary>
+        /// <param name="orderId">The Id of the order</param>
+        /// <param name="symbol">Asset Symbol</param>
+        /// <param name="time">Date/time of this event</param>
+        /// <param name="fillPrice">Fill price</param>
+        /// <param name="fillQuantity">Fill quantity</param>
+        /// <param name="orderFee">The order fee</param>
+        public OrderEvent(int orderId, string symbol, DateTime time, decimal fillPrice, int fillQuantity, decimal orderFee)
```

Comment:
I had already merged the two constructors in the last commit, they both have the two new parameters. I still have to add the utcTime parameter to the OrderEvent(Order) constructor.